vsketch_cli/cli.py

```diff
@@ -180,7 +180,14 @@ def _parse_seed(seed: str) -> Tuple[int, int]:
     envvar="VSK_FULLSCREEN",
     help="Display the viewer fullscreen on the second screen if available.",
 )
-def run(target: Optional[str], editor: Optional[str], fullscreen: bool) -> None:
+@click.option(
+    "--output-dir",
+    envvar="VSK_OUTPUT_DIR",
```

Comment:
Here you want to specify a properly parametrised `click.Path` as type, so that click checks for path existence. You can even set `path_type=pathlib.Path` so you directly get a `Path` object that you can propagate in the code (which has my preference, even though this is not always reflected in the current code).

Reply:
Agreed, that's much cleaner, thanks for the tip!